src/bfc.c

```diff
@@ -175,9 +175,11 @@ bool bfc_compile_node(bfc_compiler* compiler, bfast_node_t* node) {
         }
 
         // Read a byte from STDIN
-        // Unimplemented right now
         case ',': {
-          return false;
+          EMIT("push t_register, bf_ptr");
+          EMIT("push t_syscall, sys_read");
+          EMIT("syscall");
```

Comment:
That's not gonna work, by this you overwrite the bf_ptr register with whatever key the user inputted. You'd want to copy the input byte to a special register first and then push it onto the brainfuck-stack.